docs/guides/deploy-guide/manager.md

```diff
@@ -167,41 +180,42 @@ This is recommended.
    ./run.sh reboot
    ```
 
-### Step 4: Deploy the manager service
+### Step 5: Deploy the manager service
 
-1. Transfer the configuration repository.
+1. Push your local configuration repo state of your seed node to to the git server
```

Comment:
This is already part of "Creating a new configuration repository" and is not necessary here. There is a pointer to the "Creating a new configuration repository" chapter at the beginning of this chapter.

Reply:
I agree, removed it